### ISSUE TITLE
Cannot read properties of undefined (reading 'clerkJSUrl')

### ISSUE THREAD

**Issue by PaulieScanlon** [panteliselef/astro-with-clerk-auth#119]
Hi, i wanted to try out the integration but after following [this](https://clerk.com/changelog/2024-03-28) and [this](https://github.com/panteliselef/astro-with-clerk-auth/blob/main/packages/astro-clerk-auth/README.md) i'm seeing the following error:

```shell
$ astro dev
15:44:41 [vite] Error when evaluating SSR module /Users/paulscanlon/Documents/Paulie/work/Netlify/development/netlify-astro-clerk-astrodb/astro.config.mjs:
|- TypeError: Cannot read properties of undefined (reading 'clerkJSUrl')
    at Module.<anonymous> (file:///Users/paulscanlon/Documents/Paulie/work/Netlify/development/netlify-astro-clerk-astrodb/node_modules/astro-clerk-auth/dist/chunk-WIIW4CNN.js:11:31)
    at /Users/paulscanlon/Documents/Paulie/work/Netlify/development/netlify-astro-clerk-astrodb/astro.config.mjs:7:23
    at async instantiateModule (file:///Users/paulscanlon/Documents/Paulie/work/Netlify/development/netlify-astro-clerk-astrodb/node_modules/vite/dist/node/chunks/dep-whKeNLxG.js:55071:9)

[astro] Unable to load your Astro config

Cannot read properties of undefined (reading 'clerkJSUrl')
  Stack trace:
    at Module.<anonymous> (file:///Users/paulscanlon/Documents/Paulie/work/Netlify/development/netlify-astro-clerk-astrodb/node_modules/astro-clerk-auth/dist/chunk-WIIW4CNN.js:11:31)
    at async instantiateModule (file:///Users/paulscanlon/Documents/Paulie/work/Netlify/development/netlify-astro-clerk-astrodb/node_modules/vite/dist/node/chunks/dep-whKeNLxG.js:55071:9)
error Command failed with exit code 1.
```

## Reproduction Repo

here's the reproduction repo [netlify-astro-clerk-astrodb](https://github.com/PaulieScanlon/netlify-astro-clerk-astrodb) my `.env` has the same as shown in the `.env.example` file.

Thanks in advance!

**Comment by panteliselef** [comment]
@PaulieScanlon Thanks reporting this. While I investigate what's wrong with the latest version, could u try installing `astro-clerk-auth@0.2.0` and check if that works for you ?

**Comment by panteliselef** [comment]
@PaulieScanlon Sorry for having to experience this issue. This is now fixed in `astro-clerk-auth@0.3.2`. Feel free to close this issue once you have verified the fix

**Comment by panteliselef** [comment]
This issue is considered fixed, i'm closing due to inactivity
